runtime/lib/app-runtime.js

```diff
@@ -1265,6 +1265,10 @@ AppRuntime.prototype.onLoggedIn = function onLoggedIn (config) {
     logger.error('Unexpected error on logged in', err.stack)
     return onDone()
   }).then(this.startDaemonApps.bind(this))
+    .then(() => {
+      // reset bootPriority
+      os.setPriority(0)
+    })
```

Comment:
Shall we emit an event just like ready, or done, and set the priority at "services/vuid/index.js"? It makes the runtime still be pure enough :)